pkg/client/cli/license.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+)
+
+type licenseInfo struct {
+	outputFile string
+}
+
+func LicenseCommand() *cobra.Command {
+	li := &licenseInfo{}
+	cmd := &cobra.Command{
+		Use:  "license",
+		Args: cobra.MinimumNArgs(1),
+
+		Short: "Get License from Ambassador Cloud",
+		Long:  "Get License from Ambasssador Cloud",
+		RunE:  li.getCloudLicense,
+	}
+	flags := cmd.Flags()
+
+	flags.StringVarP(&li.outputFile, "output-file", "f", "/tmp/license", "The file where you want the license secret to be output to")
```

Comment:
Use `filepath.Join(os.TempDir(), "license")` instead of hard-coding `/tmp`.

Reply:
Mostly because of `curl` using `-o` for output and `kubectl apply` using `-f` for input; I'd expect the short flag here to be `-o` and not `-f`.

Reply:
Or maybe have the default be to dump it to stdout instead of using `os.TempDir()`.